### ISSUE TITLE
<!DOCTYPE html> is stripped out

### ISSUE THREAD

**Issue by nemtsov** [chrisvfritz/prerender-spa-plugin#20]
From `examples/generic-simple`:

input: 

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Generic &amp; Simple Prerendering Demo</title>
  </head>
  <body>
    <input id="new-todo" placeholder="New todo">
    <div id="todos"></div>
    <script src="main.js"></script>
  </body>
</html>
```

output:

```html
<html><head>
    <title>Generic &amp; Simple Prerendering Demo</title>
  </head>
  <body>
    <input id="new-todo" placeholder="New todo">
    <div id="todos"><ul><li class="item">Do the dishes <button class="remove-todo" data-index="0">X</button></li><li class="item">Make the bed <button class="remove-todo" data-index="1">X</button></li><li class="item">Take out the trash <button class="remove-todo" data-index="2">X</button></li></ul></div>
    <script src="main.js"></script>
  

</body></html>
```

Note that the `<!DOCTYPE html>` is stripped out.
It is important that the doctype is preserved, as it affects page rendering.

**Comment by qgustavor** [comment]
I was just going to post this issue, but I'm one hour delayed... Seems the problem is [on this line](https://github.com/chrisvfritz/prerender-spa-plugin/blob/81f5f75f7ef55b4ef689691536a2a3e25e8fe964/lib/phantom-page-render.js#L77): `return document.documentElement.outerHTML`. Isn't hard [to get the doctype](https://stackoverflow.com/questions/6088972/get-doctype-of-an-html-as-string-with-javascript) and fix that.

**Comment by chrisvfritz** [comment]
Good catch! I'll add this fix as soon as I can given the holidays.

**Comment by chrisvfritz** [comment]
Fixed in the [2.0 release](https://github.com/chrisvfritz/prerender-spa-plugin/releases/tag/v2.0.0). 🎉
